miden/src/cli/data.rs

```diff
@@ -175,10 +196,16 @@ impl ProgramFile {
         let now = Instant::now();
 
         // compile program
-        let program = Assembler::default()
+        let mut assembler = Assembler::default()
             .with_library(&StdLibrary::default())
             .map_err(|err| format!("Failed to load stdlib - {}", err))?
-            .with_debug_mode(debug)
+            .with_debug_mode(debug.is_on());
```

Comment:
minor nit: maybe change the order of calls so that `map_error` is the last one. For example:
```Rust
let mut assembler = Assembler::default()
    .with_debug_mode(debug.is_on())
    .with_library(&StdLibrary::default())
    .map_err(|err| format!("Failed to load stdlib - {}", err))?;
```